src/app/task/shared/models/task.model.ts

```diff
@@ -1,11 +1,12 @@
 export class Task {
```

Comment:
Is there a reason to make it a class? Maybe it can be an interface or a type. Please have a look where to use classes, interfaces and types.